exporter/importables.go

```diff
@@ -896,4 +903,68 @@ var resourcesMap map[string]importable = map[string]importable{
 			return nil
 		},
 	},
+	"databricks_notebook": {
+		Service: "notebooks",
+		Name: func(d *schema.ResourceData) string {
+			name := d.Get("path").(string)
+			if name == "" {
+				return d.Id()
+			} else {
+				name = strings.TrimPrefix(name, "/")
+			}
+			re := regexp.MustCompile(`[^0-9A-Za-z_]`)
+			return strings.ToLower(re.ReplaceAllString(name, "_"))
```

Comment:
isn't it done already by normalizer in the context? it was your comment yesterday ;-)

Reply:
ToLower definitely not done, good idea to move it there tomorrow 😜

Reply:
We need to remove that explicit normalization from more places, I see it in `databricks_global_init_script` & `databricks_repo`